src/main/java/marquez/client/MarquezClient.java

```diff
@@ -62,15 +62,15 @@
   @VisibleForTesting final MarquezHttp http;
 
   public MarquezClient() {
-    this(DEFAULT_BASE_URL);
+    this(DEFAULT_BASE_URL, null);
   }
 
-  public MarquezClient(final String baseUrlString) {
-    this(Utils.toUrl(baseUrlString));
+  public MarquezClient(final String baseUrlString, @Nullable final String apiKey) {
+    this(Utils.toUrl(baseUrlString), apiKey);
   }
 
-  public MarquezClient(final URL baseUrl) {
-    this(MarquezUrl.create(baseUrl), MarquezHttp.create(MarquezClient.Version.get()));
+  public MarquezClient(final URL baseUrl, @Nullable final String apiKey) {
+    this(MarquezUrl.create(baseUrl), MarquezHttp.create(MarquezClient.Version.get(), apiKey));
```

Comment:
Should we also keep constructors with just the URL as parameter? Since this is the default use in Marquez.

Reply:
Ahh, very good point! Yep, will add single arg constructor accepting only `URL` as well.